docs/docs/terms.md

```diff
@@ -0,0 +1,34 @@
+---
+id: terms
+title: Glossary of Terms
+sidebar_label: Terms
+---
+
+### Host
+
+The device that will receive the key strokes from the keyboard
+
+### Central
```

Comment:
Should these both be explicit about the context of a split keyboard?

Reply:
I can do that. I was hoping to make it clear that the central is the main part of the keyboard so it could be used throughout the docs as the.. well central.

Reply:
So.. the problem is, for non splits.... It's primary purpose is to be a peripheral to the host. Hmmm....

Reply:
How can we make it more clear that the central is the host's peripheral and peripheral is the central's?

Reply:
Perhaps the term "host peripheral" has meaning here? It's a peripheral to a host, both for the non-split and split scenario. *And* in the split case that same side is also a "split central"?

Reply:
something like I've got now?